brian_cheung/week_two_challenges/more_methods.md

```diff
@@ -64,11 +96,18 @@ big_ordered_array(test_array)
 # output => ['I', 'OF', 'FULL', 'LOVE', 'ARRAYS', 'MAKING', 'STRINGS']
 ```
 ```ruby
-# write your code here
+def big_ordered_array (array)
+  puts array.map {|word| word.upcase }.sort_by { |word| word.length }.group_by(&:length).map{ |itterate, array| array.sort}.flatten
+end
```

Comment:
What's this all about? `.map{ |itterate, array| array.sort}`

Reply:
iterate goes to each hash and the array passes in the hash to be sorted alphabetically with .sort then flatten merges them all into one array